packages/amplify-graphql-schema-generator/src/__tests__/generate-ts-data-schema.test.ts

```diff
@@ -7,19 +7,19 @@ jest.mock('../utils', () => ({
   getHostVpc: jest.fn(() => {
     return {
       vpcId: 'abc',
-      securityGroupIds: ['sb0', 'sb1', 'sb2'],
+      securityGroupIds: ['sg0', 'sg1', 'sg2'],
       subnetAvailabilityZoneConfig: [
         {
-          subnetId: 'sb1',
-          availabilityZone: 'az1',
+          subnetId: 'sg0',
```

Comment:
subnets can stay "sb" -- i was trying to say that we needed to disambiguate your identifiers so that we don't confuse security group IDs with subnet IDs. :)